app/web/i18n/constants.ts

```diff
@@ -0,0 +1,99 @@
+interface Language {
+  name: string;
+  flagIconCode: string;
+}
+
+interface LanguageMap {
+  [key: string]: Language;
+}
+
+export const LANGUAGE_MAP: LanguageMap = {
+  cs: {
+    name: "Czech",
+    flagIconCode: "CZ",
+  },
+  de: {
+    name: "German",
+    flagIconCode: "DE",
+  },
+  en: {
+    name: "English",
+    flagIconCode: "GB",
+  },
+  es: {
+    name: "Spanish",
+    flagIconCode: "ES",
+  },
+  "es-419": {
+    name: "Spanish",
+    flagIconCode: "MX",
+  },
+  fr: {
+    name: "French",
+    flagIconCode: "FR",
+  },
+  "fr-CA": {
+    name: "French (Canada)",
+    flagIconCode: "CA",
+  },
+  hi: {
+    name: "Hindi",
+    flagIconCode: "IN",
+  },
+  it: {
+    name: "Italian",
+    flagIconCode: "IT",
+  },
+  ja: {
+    name: "Japanese",
+    flagIconCode: "JP",
+  },
+  "nb-NO": {
+    name: "Norwegian (Bokm?l) (Norway)",
```

Comment:
I think you want å here instead of the questionmark? I don't think you need the Norway in parens since it already says Norwegian.